internships/Outreachy_2023-05/2023-07-20_meeting.md

```diff
@@ -0,0 +1,53 @@
+# Outreachy meeting
+
+### scikit-image project: narrative documentation
+
+- **Time:** Thursday, Jul 20, 2023, 17:00–17:30 UTC
+- **[Join video call via Zoom](https://us06web.zoom.us/j/88060567580?pwd=THRpaWFnSFNwK0Fycy9FVk5RYnV5UT09)**
+- **Notes:** https://hackmd.io/@mkcor/outreachy-meeting
+- **[Archive](https://github.com/scikit-image/skimage-archive/internships/Outreachy_2023-05/)**
+
+**Present:** Ananya, Marianne
+
+**Moderator:** Marianne
+
+
+## Agenda
+
+### Review of the week
+
+- Week 7: [Timeline](https://hackmd.io/@mkcor/outreachy-roadmap)
+    - It's too early to submit a PR!
+    - Getting a hold of the data hasn't been trivial.
+    - Didn't expect this (to this extent). 
+- Got started with `pyklb`.
+    - Reads KLB files into NumPy arrays.
+    - Works at least in a minimal, 'clean' environment.
+    - If needed, will save and load the arrays.
+
+### Career opportunities
+
+- Outreachy's theme for Week 7.
+- Ananya is looking for research internships in academia.
+- Ananya will ask Marianne for recommendation letters when she applies.
+- Open source + open science
+- [DOI](https://www.doi.org/)s aren't just for papers (see [Zenodo](https://zenodo.org/)).
+- See also [ORCiD](https://orcid.org/).
+
+### 2D segmentation
+
+- [Segment human cells](https://scikit-image.org/docs/stable/auto_examples/applications/plot_human_mitosis.html)
+- [Thresholding guide](https://scikit-image.org/docs/stable/auto_examples/applications/plot_thresholding_guide.html)
+- [A tutorial on segmentation](https://emmanuelle.github.io/a-tutorial-on-segmentation.html)?
+
+### Image license
+
+- Origin: https://idr.openmicroscopy.org/webclient/?show=project-502
+- License: CC BY 4.0
+- Ask for relicensing under CC0?
+- Or search for similar datasets already under CC0?
```

Comment:
@joshmoore how can I search for given licenses in the IDR, please?

Reply:
There's a drop down on https://idr.openmicroscopy.org. If you choose, "License" you can choose, CC-BY, CC0, etc. For example, for CC0:

https://idr.openmicroscopy.org/search/?key=License&value=CC0+1.0+https%3A%2F%2Fcreativecommons.org%2Fpublicdomain%2Fzero%2F1.0%2F&resource=container&operator=contains

<img width="1745" alt="image" src="https://github.com/scikit-image/skimage-archive/assets/88113/5d4020f7-a867-41e1-b4a9-0253fa0933a0">